pkg/operator/controller/externaldns/pod.go

```diff
@@ -60,6 +60,9 @@ const (
 	awsAccessKeySecretEnvVar = "AWS_SECRET_ACCESS_KEY"
 	awsAccessKeyIDKey        = "aws_access_key_id"
 	awsAccessKeySecretKey    = "aws_secret_access_key"
+	awsSharedCredentialsFile = "AWS_SHARED_CREDENTIALS_FILE"
+	awsCredentialsVolumeName = "aws-credentials-file"
+	awsCredentialsFileKey    = "aws-credentials.json"
```

Comment:
The credentials file is a not a JSON file. Any reason you gave it a JSON file suffix?

Reply:
Removed the `.json` suffix